public/config.json

```diff
@@ -1,5 +1,10 @@
 {
-  "apiBase": "/client/api",
+  "servers": [
+    {
+      "name": "Manager 1",
```

Comment:
This may not always work; due to CORS if they are on different domain and using https.

Reply:
@rhtyd Will it work well if I use the baseUrl (domain, https) instead of the current one?

Reply:
@rhtyd what do you think about my way?